### ISSUE TITLE
Question: escaping "#" from HexColorInput

### ISSUE THREAD

**Issue by atomcorp** [omgovich/react-colorful#62]
Hey, final one from me I think 😂

I was just wondering about the decision to remove the "#" when setting the HexColorInput's local state. My use case is to avoid confusion for the less tech-savvy when they copy/paste the input. There doesn't appear a way to force prefixing it.

I would think the obvious solution would be to add a boolean prop like "showHex" to toggle it.

Not a deal breaker for me personally, but I wanted to hear you thoughts. 

(Sorry for all the problems but none of the contributions, I'll have a pop at some of this if I find some time this week)

Cheers

**Comment by atomcorp** [comment]
I guess on a somewhat related note is that HexColorInput won't accept a copy / pasted hex value like `#FF4136`. 

I think that's okayish if the input doesn't show the "#", but it would probably need to work if the above suggestion was added.

**Comment by atomcorp** [comment]
Been looking at this a bit closer, I didn't realise how trivial it is to just use my own <input /> component. I just sort of assumed HexColorInput was necessary.

 I think it's probably cool to ignore my suggestion 😅

**Comment by omgovich** [comment]
Hey @atomcorp! 
Your issue helped me to understand that if user pastes a color that starts with `#` we don't process it properly (lose the last digit of the color):
```
#AABBCC → CMD+V → value="AABBC"
```
I just released the 4.0.6 and with improved pasting experience:
```
#AABBCC → CMD+V → value="AABBCC"
```
